### DISCUSSION TITLE
Share via LocalSend on Android always shares previous file

### DISCUSSION THREAD

**Discussion by Annihilannic** [localsend/localsend#1502] [Discussion (English, International)]
I love this App, I find it most useful for transferring recently taken photographs from Android phone to Windows 10 PC.

However a minor annoyance with this is when I locate a photo on Android, choose 'share' icon, select LocalSend, it often has the last file I sent selected (could be from days or weeks ago!) and not the one I just located.  I have to delete the old one from the shared files, go back, share again, etc.

I'm not sure whether this is a feature of the App or the OS - anyone else seeing this?  

(Android 14, Samsung Galaxy A52)

**Comment by Tienisto** [comment]
You probably have "Multiple Recipients" selected. By enabling "Single Recipient", the selection gets cleared after a successful transfer.

**Comment by Annihilannic** [comment]
Thanks for the suggestion... took me a while to find the option, but it seems I already have it set to "Single recipient".

**Comment by Annihilannic** [comment]
It only seems to happen to the first file I share after the app has been asleep for a long time.  Subsequent share attempts are immediately refreshed with the new file.  Feels like a Samsung app sleeping "feature" maybe?

**Comment by margorpi** [comment]
I also use a Samsung phone (S23U) and encountered the same issue, so I had to search for solutions and ended up here. 

When I select a file from an app, such as the phone's file manager or photo album, and share it to LocalSend via Android's share function, the target computer always receives the previously shared file (it seems the filename is always "1"). Then, I have to share again to send the current file successfully.

**Comment by I21b** [comment]


**Reply by margorpi** [reply]
Not that case.

**Comment by iamLazyCode** [comment]
is there any fix for this ? facing the same isue
